frontend/src/pages/Alerts/LogAlert/LogAlertPage.tsx

```diff
@@ -139,7 +135,7 @@ export const LogAlertPage = () => {
 				query: data?.log_alert.query,
 				name: data?.log_alert.Name,
 				belowThreshold: data?.log_alert.BelowThreshold,
-				excludedEnvironments: data?.log_alert.ExcludedEnvironments,
+				excludedEnvironments: [], // log alerts use query instead of excluded environments
```

Comment:
What's the reason for including this at all? Is it just because the `LogAlertForm` inherits from `AlertForm`? If yes, I think could we use something like `Omit<AlertForm, 'excludedEnvironments'>` to update the types and drop this.